blocks/hooks/anchor.js

```diff
@@ -34,9 +34,6 @@ export function addAttribute( settings ) {
 		settings.attributes = assign( settings.attributes, {
 			anchor: {
 				type: 'string',
-				source: 'attribute',
```

Comment:
Why is this attributes definition modified?

Reply:
>This change allows the server to have access to anchor attribute which is required to generate the styles in the server.

This will also mean existing blocks with anchors will lose the attribute value after this change (since it was not encoded in the comment delimiters).

Reply:
I see this was mentioned in https://github.com/WordPress/gutenberg/pull/3722#discussion_r155362853